examples/examples/apexLogStream/apexLogStream.ts

```diff
@@ -30,9 +30,8 @@ export async function run() {
     });
   }
 
-  // List all auth files
-  const authFiles = await AuthInfo.listAllAuthFiles();
-  const orgs = authFiles.map(authfile => authfile.replace('.json', ''));
+  const config = await GlobalInfo.getInstance();
+  const orgs = Object.keys(config.authorizations)
```

Comment:
I think we need a `type` on authorizations. For example, what about the functions account? So this would be something like `config.authorizations.filter(auth => auth.type === 'org')` or something like that. OR maybe something like the following:

```
// eventually you could have 'heroku', 'globalId' etc.
type AuthorizationType = 'org' | 'functions'; 
...
// If nothing is passed in, return all of them
config.getAuthorizations(type?: AuthorizationType)
```

OR. Maybe this is through environment?
```
// eventually you could have 'heroku', 'globalId' etc. Maybe enum instead.
type AuthorizationType = 'org' | 'functions';
type EnvironmentType = 'org' | 'compute';
// If there isn't a token on the env, or they need to refresh the token, it would have to go to the authorization.
...
// If nothing is passed in, return all of them
config.getEnvironments(type?: EnvironmentType)
```

Just some thoughts.